src/screens/Settings/components/Tools/index.tsx

```diff
@@ -47,9 +47,10 @@ export default function Tools({ wineVersion, winePrefix }: Props) {
       // Use DataTransferItemList interface to access the file(s)
       // If dropped items aren't files, reject them
       if (ev.dataTransfer.items[0].kind === 'file') {
-        const exe = ev?.dataTransfer?.items[0]?.getAsFile()?.path
-
-        return callTools('runExe', exe)
+        const exe = ev.dataTransfer.items[0].getAsFile()?.path
```

Comment:
What happens if `dataTransfer` comes empty?

Reply:
check the if before:  
`if( env.dataTransfer.items)`
We already catched that.

Reply:
ok then

Reply:
i found that while testing. I could never reach the point, because i can't manipulate the dataTransfer after the if statement in time.